src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.OpenTextBufferManager.cs

```diff
@@ -269,8 +269,8 @@ internal void Disconnect(bool documentIsClosed, bool rollbackTemporaryEdits)
             {
                 AssertIsForeground();
 
-                // Detatch from the buffer; it is important that this is done before we start
-                // undoing transactions, since the undos will cause buffer changes.
+                // Detach from the buffer; it is important that this is done before we start
+                // undoing transactions, since the undo's will cause buffer changes.
```

Comment:
I think "undos" is the plural form of "undo", and neither possessive nor a contraction, so it shouldn't have an apostrophe.
Consider "undo actions"